tests/functional/test_tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+envlist = py26,py27,pypy,py32,py33,py34,pypy3
```

Comment:
if possible, I would for later reader keep the version in the same order. Sometime pypy is before Python2, sometime both pypy and pypy3 are at the end.